scripts/generator/graphql_generator/csharp/SDK/DesktopMessageReceiver.cs.erb

```diff
@@ -0,0 +1,56 @@
+namespace <%= namespace %>.SDK {
+    using System;
+    using Shopify.Unity.SDK;
+    using Shopify.Unity.GraphQL;
+
+    #if !SHOPIFY_MONO_UNIT_TEST
+    using UnityEngine;
+    public partial class DesktopMessageReceiver: MonoBehaviour {}
+    #endif
+
+    public partial class DesktopMessageReceiver {
```

Comment:
> Introduce SetupDesktopMessageReceiver which sets up the DesktopMessageReceiver. Wondering now if there is maybe a base class between the NativeWebViewMessageReceiver and this new class?

Yea I can see that too. They pretty much do the same thing except the name of the method that invoked to do the validation check. I can see moving this logic to an abstract base, wrapping the validation in a method and calling it from the subclasses